### PR TITLE
Create pl.json

### PR DESCRIPTION
**Pull request by nepozs**
Added polish translation.

### CODE REVIEW

File: custom_components/blitzortung/translations/pl.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Blitzortung",
+        "description": "Konfiguracja wykrywania piorunów Blitzortung",
```
Comment (bieniu) [inline custom_components/blitzortung/translations/pl.json]:
```suggestion
        "description": "Konfiguracja wykrywania błyskawic Blitzortung",
```

File: custom_components/blitzortung/translations/pl.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    "error": {
+    },
+    "abort": {
+    }
+  },
+  "options": {
+    "step": {
+      "init": {
+        "title": "Opcje Blitzortung",
+        "description": "Konfiguracja wykrywania piorunów Blitzortung",
```
Comment (bieniu) [inline custom_components/blitzortung/translations/pl.json]:
```suggestion
        "description": "Konfiguracja wykrywania błyskawic Blitzortung",
```

File: custom_components/blitzortung/translations/pl.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        "data": {
+          "radius": "Promień wykrywania wyładowań atmosferycznych (km)",
```
Comment (bieniu) [inline custom_components/blitzortung/translations/pl.json]:
```suggestion
          "radius": "Promień wykrywania błyskawic (km)",
```

File: custom_components/blitzortung/translations/pl.json [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+          "latitude": "Szerokość geograficzna",
```
Comment (bieniu) [inline custom_components/blitzortung/translations/pl.json:25]:
```suggestion
          "idle_reset_timeout": "Limit czasu resetowania bezczynności (minuty, 0 - wyłączony)",
          "latitude": "Szerokość geograficzna",
```

**Comment by mrk-its** [comment]
Merged, thanks!

**Pull request merged**